connector/src/main/java/org/geysermc/connector/network/session/cache/EntityCache.java

```diff
@@ -49,6 +50,7 @@
     private Long2LongMap entityIdTranslations = Long2LongMaps.synchronize(new Long2LongOpenHashMap());
     private Map<UUID, PlayerEntity> playerEntities = Collections.synchronizedMap(new HashMap<>());
     private Object2LongMap<UUID> bossbars = new Object2LongOpenHashMap<>();
+    private Int2ObjectOpenHashMap<Integer[]> cachedEntityLinks = new Int2ObjectOpenHashMap<>();
```

Comment:
Should be `private Int2ObjectMap<int[]> cachedEntityLinks = new Int2ObjectOpenHashMap<>()`.